### DISCUSSION TITLE
[QUESTION]: Mobile devices webview

### DISCUSSION THREAD

**Discussion by simplecommerce** [cuongdevjs/reactjs-social-login#151] [Q&A] [answered]
Hi,

Does this work on mobile apps that are built using webviews?

I did a quick test on appmysite.com and it seems to work on IOS but on Android logins like Facebook says it doesn't allow to login.
Also, in some cases Google login doesn't always work.
I click and nothing happens.

**Comment by cuongdevjs** [comment, marked as answer]
I am unsure about this, I haven't tested this case previously. But in webview, the environment is the same, in my opinion, you will need to set some additional configures on Android/Ios
